association_backend/association/domain/entities/stripe_entities.py

```diff
@@ -0,0 +1,43 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import Optional
+
+from pydantic import validator
+from pydantic.main import BaseModel
+
+
+@dataclass
+class StripeCustomer:
+    id: str
+    email: str
+
+
+@dataclass
+class StripeCheckoutSession:
+    id: str
+    customer_id: Optional[str] = ""
+    subscription_id: Optional[str] = ""
+
+
+class StripeCheckoutSessionInput(BaseModel):
+    customer_id: Optional[str] = None
+    customer_email: Optional[str] = None
```

Comment:
@patrick91 what do you think about those stripe entities? Not a big fan, I think we should hide Stripe in the repository, but I don't think we should give to it entities as our business model is not around Stripe, ideally, checkout session/stripe customer are hidden as concepts?